docs/en/03-benchmark/supported_models.md

```diff
@@ -2,79 +2,79 @@
+| ESRGAN                      | MMEditing        |      Y      |      Y      |    Y     |  Y   |   Y   |    Y     |  N   |     [config](https://github.com/open-mmlab/mmediting/tree/master/configs/restorers/esrgan)      |
+| SRGAN                       | MMEditing        |      Y      |      Y      |    Y     |  Y   |   Y   |    Y     |  N   | [config](https://github.com/open-mmlab/mmediting/tree/master/configs/restorers/srresnet_srgan)  |
+| SRResNet                    | MMEditing        |      Y      |      Y      |    Y     |  Y   |   Y   |    Y     |  N   | [config](https://github.com/open-mmlab/mmediting/tree/master/configs/restorers/srresnet_srgan)  |
+| Real-ESRGAN                 | MMEditing        |      Y      |      Y      |    Y     |  Y   |   Y   |    Y     |  N   |   [config](https://github.com/open-mmlab/mmediting/tree/master/configs/restorers/real_esrgan)   |
+| EDSR                        | MMEditing        |      Y      |      Y      |    Y     |  Y   |   N   |    Y     |  N   |      [config](https://github.com/open-mmlab/mmediting/tree/master/configs/restorers/edsr)       |
+| RDN                         | MMEditing        |      Y      |      Y      |    Y     |  Y   |   Y   |    Y     |  N   |       [config](https://github.com/open-mmlab/mmediting/tree/master/configs/restorers/rdn)       |
+| DBNet                       | MMOCR            |      Y      |      Y      |    Y     |  Y   |   Y   |    Y     |  N   |          [config](https://github.com/open-mmlab/mmocr/tree/main/configs/textdet/dbnet)          |
+| PANet                       | MMOCR            |      Y      |      Y      |    Y     |  Y   |   ?   |    Y     |  N   |          [config](https://github.com/open-mmlab/mmocr/tree/main/configs/textdet/panet)          |
+| DBNet                       | MMOCR            |      Y      |      Y      |    Y     |  Y   |   ?   |    Y     |  N   |         [config](https://github.com/open-mmlab/mmocr/tree/main/configs/textdet/psenet)          |
+| CRNN                        | MMOCR            |      Y      |      Y      |    Y     |  Y   |   Y   |    N     |  N   |         [config](https://github.com/open-mmlab/mmocr/tree/main/configs/textrecog/crnn)          |
+| SAR                         | MMOCR            |      N      |      Y      |    N     |  N   |   N   |    N     |  N   |          [config](https://github.com/open-mmlab/mmocr/tree/main/configs/textrecog/sar)          |
+| SATRN                       | MMOCR            |      Y      |      Y      |    Y     |  N   |   N   |    N     |  N   |         [config](https://github.com/open-mmlab/mmocr/tree/main/configs/textrecog/satrn)         |
+| HRNet                       | MMPose           |      N      |      Y      |    Y     |  Y   |   N   |    Y     |  N   |     [config](https://mmpose.readthedocs.io/en/latest/papers/backbones.html#hrnet-cvpr-2019)     |
```

Comment:
hrnet is not tested or really not supported?

Reply:
only supports mmcls and mmseg and yolos